t/sys/write-read.c

```diff
@@ -88,7 +88,7 @@ int write_read_test(char* unifyfs_root)
 
     /* Check global and local size on our un-laminated file */
     get_size(path, &global, &local, &log);
-    ok(global == 0, "%s: global size is %d: %s",  __FILE__, global,
+    ok(global == 15, "%s: global size is %d: %s",  __FILE__, global,
```

Comment:
I thought we decided that un-laminated files always reported size 0?

Reply:
I still think that's a good optimization for codes that can use it.  Since computing file size before lamination is expensive, it'd probably be nice to have an option to skip it and report 0.  However, I seem to recall that HDF requires an accurate file size at times *before* lamination.